openquake/calculators/extract.py

```diff
@@ -751,6 +751,9 @@ def extract_agg_curves(dstore, what):
     Returns an array of shape (#periods, #stats) or (#periods, #rlzs)
     """
     info = get_info(dstore)
+    # NOTE: in loss types like structural+nonstructural the '+' needs to be
+    # encoded, otherwise it is interpreted as a space by parse_qs
+    what = what.replace('+', '%2B')
```

Comment:
You want the function `urllib.parse.quote_plus`

Reply:
Done